MonoGame.Framework/Graphics/SpriteBatch.cs

```diff
@@ -168,16 +171,17 @@ void CheckValid(SpriteFont spriteFont, StringBuilder text)
         /// Submit a sprite for drawing in the current batch.
         /// </summary>
         /// <param name="texture">A texture.</param>
-        /// <param name="position">An optional drawing location on screen.</param>
-        /// <param name="destinationRectangle">An optional drawing bound.</param>
+        /// <param name="position">The drawing location on screen or null if <param name="destinationRectangle"> is used.</param></param>
+        /// <param name="destinationRectangle">The drawing bounds on screen or null if <param name="position"> is used.</param></param>
         /// <param name="sourceRectangle">An optional region on the texture which will be rendered. If null - draws full texture.</param>
-        /// <param name="origin">An optional center of rotation.</param>
-        /// <param name="rotation">An optional rotation of this sprite.</param>
-        /// <param name="scale">An optional scale vector.</param>
-        /// <param name="color">An optional color mask.</param>
-        /// <param name="effects">The optional drawing modificators.</param>
-        /// <param name="layerDepth">An optional depth of the layer of this sprite.</param>
+        /// <param name="origin">An optional center of rotation. Uses <see cref="Vector2.Zero"/> if null.</param>
+        /// <param name="rotation">An optional rotation of this sprite. 0 by default.</param>
+        /// <param name="scale">An optional scale vector. Uses <see cref="Vector2.One"/> if null.</param>
+        /// <param name="color">An optional color mask. Uses <see cref="Color.White"/> if null.</param>
+        /// <param name="effects">The optional drawing modificators. <see cref="SpriteEffects.None"/> by default.</param>
+        /// <param name="layerDepth">An optional depth of the layer of this sprite. 0 by default.</param>
         /// <exception cref="InvalidOperationException">Throwns if both <paramref name="position"/> and <paramref name="destinationRectangle"/> been used.</exception>
+        /// <remarks>This overload uses optional parameters.</remarks>
         /// <remarks>This overload requires only one of <paramref name="position"/> and <paramref name="destinationRectangle"/> been used.</remarks>
```

Comment:
Note i'm not sure if having two `<remarks>` blocks is valid doc format or how it should behave.  I know currently SharpDoc is ignoring the second remark block.  I can fix that, but we need to be sure what we're doing is valid.